setup.py

```diff
@@ -183,7 +183,13 @@ def run(self):
 with open('VERSION_NUMBER') as f:
     version_number = f.readline().strip()
 if nightly_build:
-    date_suffix = str(datetime.datetime.now().date().strftime("%m%d"))
+    #https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables
+    date_suffix = environ.get('BUILD_BUILDNUMBER')
+    if date_suffix is None:
+      #The following line is only for local testing
+      date_suffix = str(datetime.datetime.now().date().strftime("%m%d"))
```

Comment:
>strftime("%m%d")) [](start = 55, length = 17)

Need to fix this as well?

Reply:
It's a fallback path. This line is only for only for local testing. On our build machines, BUILD_BUILDNUMBER is always set.

Reply:
So, why don't we make the fallback include the year ?

---
In reply to: [362934419](https://github.com/microsoft/onnxruntime/pull/2771#discussion_r362934419) [](ancestors = 362934419)

Reply:
Fixed.